SalesforceReact.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+
+  s.name         = "SalesforceReact"
+  s.version      = "7.0.0"
+  s.summary      = "Salesforce Mobile SDK for iOS - SalesforceReact"
+  s.homepage     = "https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative"
+
+  s.license      = { :type => "Salesforce.com Mobile SDK License", :file => "LICENSE" }
+  s.author       = { "Wolfgang Mathurin" => "wmathurin@salesforce.com" }
+
+  s.platform     = :ios, "11.0"
+
+  s.source       = { :git => "https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git",
+                     :tag => "v#{s.version}",
+                     :submodules => false }
+  
+  s.requires_arc = true
+  s.default_subspec  = 'SalesforceReact'
+
+  s.subspec 'SalesforceReact' do |salesforcereact|
+
+      salesforcereact.dependency 'React'
+      salesforcereact.dependency 'SmartSync'
+      salesforcereact.dependency 'SmartStore'
```

Comment:
Do you need `SmartStore` and `SalesforceSDKCore`? Wouldn't they get pulled in automatically as transitive dependencies through `SmartSync`?

Reply:
I didn't change this file. That is the way all our specs are setup. We could try without - I think we might have run into issues in the past.

Reply:
I think we could do without. Transitive dependencies work in Cocoapods. For example, this one doesn't have Lumberjack, because Analytics pulls it in automatically. Similarly, we removed PromiseKit from the template because SwiftSDK automatically pulled it in.

Reply:
You're probably right. I'll try it.

Reply:
Did this work?

Reply:
Shoot, never tried it. I'll try and let you know.